torso-bundle.js

```diff
@@ -305,11 +305,56 @@
           },
 
           /**
+           * Will force the cache to fetch a subset of this collection's tracked ids
+           * @method requesterMixin.fetchSubsetOfTrackedIds
```

Comment:
did you pull --rebase?

If you have, then it's possible the person before you didn't gulp bundle before merging